5_recognition.ipynb

```diff
@@ -310,18 +548,40 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "## レゴブロックの位置の取得"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Get the position of the Lego block"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "レゴブロックを抽出できる色相画像の閾値を設定します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Set the threshold of the hue image from which Lego blocks can be extracted:"
```

Comment:
Ah... the "Lego block" vs. "Lego blocks" question. XD
Is there a single block or more than one.

Reply:
Sorry for making you confused (I should have correct the Google Translate...), this part "a Lego block" (or "the (blue color) Lego block").

Reply:
Ah...  so how does this look to you:

    "Set the threshold of the hue image from which the light blue Lego block can be extracted:"

Based on previous information you provided, I inserted "the light blue Lego block".